docs: add install block to the readme

romanrostislavovich opened the pull request:
Added `install` block to the readme file.

Part of #17

xferra commented on README.md:
```diff
@@ -4,6 +4,9 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![npm package](https://img.shields.io/npm/v/@linterhub/schema.svg)](https://www.npmjs.com/package/@linterhub/schema)
 
+The schemas repo is about meta-information of the software packages, linters,
```
This is part of another PR.

romanrostislavovich committed:
docs: readme file fill

following blocks in `readme` file are fill out:
- сontribute
- installation
- usage

Part of #17

xferra reviewed (dismissed):
+1

xferra reviewed (approved):
+1
